tools/wasm-sourcemap.py

```diff
@@ -176,7 +178,7 @@ def remove_dead_entries(entries):
 
 def read_dwarf_entries(wasm, options):
   if options.dwarfdump_output:
-    output = open(options.dwarfdump_output, 'rb').read()
+    output = read_binary(options.dwarfdump_output)
```

Comment:
Maybe use `Path` directly here to keep this tool free to deps on the rest of emscripten?